app/domain/util/get_username_from_role_id.rb

```diff
@@ -0,0 +1,19 @@
+module Util
+  GetUsernameFromRoleId = CommandClass.new(
+    dependencies: {
+      role_class: ::Role
+    },
+    inputs: %i(role_id)
+  ) do
+
+    def call
+      username_from_role_id
+    end
+
+    private
+
+    def username_from_role_id
+      @role_class.username_from_roleid(@role_id)
+    end
+  end
+end
```

Comment:
Generally speaking, `CommandClass` is used to encapsulate, and make explicit, the _interaction_ of multiple dependencies.  In this case, it is simply delegating directly to `Role`, and so I think using it here is superfluous, and just adds boilerplate.  

It's being used only in `AuthenticatorStatusInput`, so I'd recommend removing this, and (for now), just grabbing the username you need in the controller, and passing it to `AuthenticatorStatusInput`, similar to my suggestion above for grabbing `role_id` in the controller.